crates/miden-lib/asm/account_components/rpo_falcon_512_procedure_acl.masm

```diff
@@ -62,10 +70,44 @@ export.auth__tx_rpo_falcon512_procedure_acl
         dup.1 neq.0
         # => [should_continue, was_any_called', i-1, pad(16)]
     end
-
     # => [was_any_called, i-1, pad(16)]
 
-    # If any trigger procedure was called, perform signature verification
+    # ------ Check if output notes were created ------
+
+    exec.tx::get_output_notes_commitment
+    # => [OUTPUT_NOTES_COMMITMENT, was_any_called, i-1, pad(16)]
+
+    padw eqw push.0 eq movdn.8 dropw dropw
+    # => [were_output_notes_created, was_any_called, i-1, pad(16)]
```

Comment:
Nit: The result of `eqw` basically says `!were_output_notes_created` so instead of `push.0 eq` I'd just use a `not` to negate it to get `were_output_notes_created`.